Mathlib/NumberTheory/NumberField/InfinitePlace/TotallyRealComplex.lean

```diff
@@ -0,0 +1,100 @@
+/-
+Copyright (c) 2022 Xavier Roblot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alex J. Best, Xavier Roblot
```

Comment:
I think that it was me who defined totally real fields because I needed them for FLT? (and then you defined totally complex fields?) I don't care at all if I'm not an author but I'm just wondering if Alex Best wrote anything in this file (I'm pretty sure Alex wouldn't care either :-) )